File: src/code-samples/langsmith/smithdb-migration/experiment-runs-query-lookup-experiment-id.ts
// :snippet-start: experiment-runs-query-lookup-experiment-id-js
import { Client } from "langsmith";

const client = new Client();
// :remove-start:
if (false) {
// :remove-end:
const experimentId = (await client.readProject({ projectName: "my-experiment" })).id;
// :remove-start:
void experimentId;
}
// :remove-end:
// :snippet-end:

File: src/code-samples/langsmith/smithdb-migration/experiment-runs-query-lookup-experiment-id.py
# :remove-start:
if False:
# :remove-end:
    # :snippet-start: experiment-runs-query-lookup-experiment-id-py
    from langsmith import Client

    client = Client()
    experiment_id = client.read_project(project_name="my-experiment").id
    # :snippet-end:
